src/main/java/fi/dy/masa/litematica/render/schematic/WorldRendererSchematic.java

```diff
@@ -668,6 +660,16 @@ public void renderEntities(Camera camera, Frustum frustum, MatrixStack matrices,
             this.world.getProfiler().swap("regular_entities");
             //List<Entity> entitiesMultipass = Lists.<Entity>newArrayList();
 
+            // TODO --> Convert Matrix4f back to to MatrixStack?
+            //  Causes strange entity behavior (translations not applied)
+            //  if this is missing ( Including the push() and pop() ... ?)
+            //  Doing this restores the expected behavior of Entity Rendering in the Schematic World
+
+            MatrixStack matrixStack = new MatrixStack();
+            matrixStack.push();
+            matrixStack.multiplyPositionMatrix(matrix4f);
+            matrixStack.pop();
+
```

Comment:
Please test and review this part of the Rendering code, I believe you recall the conversation.